### PR TITLE
Process model

### PR DESCRIPTION
**Pull request by cschroed-usgs**
There's some overlap with @isuftin's stuff here. Not sure how to proceed. Thoughts?

### CODE REVIEW

File: src/main/webapp/js/advanced/models/DataSourceModel.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+GDP.model.DataSourceVariable = Backbone.Model.extend({
+    defaults:{
+	name: null
+    }
+});
+
+GDP.model.DataSourceVariables = Backbone.Collection.extend({
+    model: GDP.model.DataSourceVariable,
+    initialize: function(models, options){
+	this.bind("add", options.view.dataSourceVariableAdd);
```
Comment (mbucknell) [inline src/main/webapp/js/advanced/models/DataSourceModel.js]: Is options.view supposed to be a Backbone.View object? I don't think it's a good practice to have a model dependent on a view. The view can use "listenTo" to list for changes in a model.

Comment (cschroed-usgs) [inline src/main/webapp/js/advanced/models/DataSourceModel.js]: Aha! jackpot! Thank you!

**Comment by mbucknell** [comment]
OCD thing: I don't like naming files with a name that is already in the directory name, i.e. ProcessModels which is in models.

I'm not sure what the overlap is with Ivan's code. I think you should talk to him

**Comment by mbucknell** [comment]
Take back previous comment about file naming. I've looked at this some more. I'm still a little confused about what is a ProcessModel. But at some point we need a model which contains all of the models that are directly associated with our views.

**Comment by isuftin** [comment]
In my mind, a ProcessModel is a description of a WPS Processing Algorithm. Carl, is that what you see it as?

**Comment by cschroed-usgs** [comment]
Ivan, I was writing it as, well to steal from Mary, "a model which contains all of the models that are directly associated with our views". Basically, it's the hub's model. I thought of the WPS Processes as 'Algorithms', but I think it's better to call them 'Processes' like you've done. Maybe we should call the hub's model a 'Job' instead -- I think it makes sense to say that as we move through the wizard we are constructing a Job to submit to the GDP.

**Comment by cschroed-usgs** [comment]
I'm going to close this as it's getting mixed up with my page-specific stuff.

**Pull request closed without merging**